### DISCUSSION TITLE
[SOLVED] Downloading glTF from glTF-export using fetch

### DISCUSSION THREAD

**Discussion by LukePetko** [iTwin/itwinjs-core#4148] [Q&A] [answered]
Hi,

I'm trying to download the glTF from glTF-export using fetch. However, I get a CORS error when trying to fetch it. Is there any way to fix this issue? 

Thanks

**Comment by mgooding** [comment]
We have not run into that issue in our testing or with our samples. Can you provide more information on your code and the error you're receiving?

**Reply by LukePetko** [reply]
The error I'm getting is ```Access to fetch at 'https://gltfprodeussa01.blob.core.windows.net/exports/...' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.``` and I'm fetching it using command: ```const data = await fetch(gltfDownloadUrl)```

**Reply by aruniverse** [reply]
~You should be passing an access token along with your request, and should have the scope required for the gltf service. Can you confirm you're doing this?~

**Reply by diegopinate** [reply]
We reproduced the issue. We are looking at potential solutions.

**Reply by diegopinate** [reply]
Just to clarify, an access token shouldn't be needed, only to use the API, the URL generated should work without one.

**Comment by diegopinate** [comment, marked as answer]
@LukePetko, we have deployed a fix for this. Could you try again and confirm if it works or if you have more issues? Thanks!

**Reply by LukePetko** [reply]
It works, thanks for the fix 🙂
